build.ps1

```diff
@@ -0,0 +1,26 @@
+# Check dotnet-format is installed or not
+$dotnetFormat = Get-Command dotnet-format -ErrorAction Ignore -CommandType Application
+
+if($dotnetFormat)
```

Comment:
nit:
```suggestion
if ($dotnetFormat)
```

Reply:
lol yeah, of note is that this is only going to be helpful for C# files.